workflows/Discord_Submission_Notify.yml

```
@@ -0,0 +1,31 @@
+name: Discord submission notify
+
+on: 
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  notify:
+    if: github.repository == 'AlternativeFFFF/Alt-F4'
+    name: Notify
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout PR
+      uses: actions/checkout@master
+    - name: Discord notification (1/2)
+      env:
+        DISCORD_WEBHOOK: ${{ secrets.DISCORD_WEBHOOK }}
+        DISCORD_USERNAME: 'ALT-F4 Bot'
+        DISCORD_AVATAR: 'https://raw.githubusercontent.com/AlternativeFFFF/Alt-F4/master/assets/GLOBAL/img/spider.png'
+      uses: Ilshidur/action-discord@master
+      with:
+        args: "New pull request {{ EVENT_PAYLOAD.pull_request.number }}, created by {{ EVENT_PAYLOAD.pull_request.user.login }}."
+    - name: Discord notification (2/2)
+      env:
+        DISCORD_WEBHOOK: ${{ secrets.DISCORD_WEBHOOK }}
+        DISCORD_USERNAME: 'ALT-F4 Bot'
+        DISCORD_AVATAR: 'https://raw.githubusercontent.com/AlternativeFFFF/Alt-F4/master/assets/GLOBAL/img/spider.png'
+      uses: Ilshidur/action-discord@master
```

As the [readme](https://github.com/Ilshidur/action-discord/blob/master/README.md) states, we should use an explicit commit SHA-1 for for stability purposes.

```suggestion
      uses: Ilshidur/actions-discord@0.2.0
```